scripts/README.md

```diff
@@ -11,32 +10,44 @@ npx truffle compile
 yarn run networks-inject
 ```
 
+Setup the env variables:
+
+```
+export PK=<Your Key>
+export GAS_PRICE_GWEI=8
+export NETWORK_NAME=<network>
+export MASTER_SAFE=<master safe>
+
+```
+
 ### Confirming multisig orders with Metamask
 
 Note that metamask underestimates the gas consumption. Just add a 4 at the beginning of the gas limit.
 
 ### Deploy Safes
+
 ```js
-truffle exec scripts/deploy_safes.js --masterSafe=0xb2162C8868AB135604270c92ed1faFA47b2BB50B --fleetSize=2 --network rinkeby
+truffle exec scripts/deploy_safes.js --masterSafe=$MASTER_SAFE --fleetSize=2 --network $NETWORK_NAME
```

Comment:
Does it make sense to have the scripts directly read the environmental variables?

Reply:
Let's keep it explicitly for now. We can change it later.